foundations/html_css/html-foundations/links-and-images.md

```diff
@@ -172,12 +172,36 @@ For example, using an absolute path we can display an image located on The Odin
 
 To use images that we have on our own websites, we can use a relative path.
 
+<details markdown="block">
+<summary class="dropDown-header">Linux, macOS, ChromeOS
+</summary>
+
 1.  Create a new directory named `images` within the `odin-links-and-images` project.
 
 2.  Next, download [this image](https://unsplash.com/photos/Mv9hjnEUHR4/download?force=true&w=640) and move it into the images directory we just created.
 
 3.  Rename the image to `dog.jpg`.
 
+</details>
+
+<details markdown="block">
+<summary class="dropDown-header">WSL
+</summary>
+
+When you download a file from the internet, Windows has a security feature that creates a hidden `Zone.Identifier` file with the same name as your downloaded file and it looks like `mypicture.jpg:Zone.Identifier` This file is harmless, but we'd like to avoid copying it over and cluttering up our directories.
+
+1.  Create a new directory named `images` within the `odin-links-and-images` project.
+
+2.  Next, download [this image](https://unsplash.com/photos/Mv9hjnEUHR4/download?force=true&w=640).
```

Comment:
```suggestion
2.  Next, [download the stock dog image](https://unsplash.com/photos/Mv9hjnEUHR4/download?force=true&w=640).
```

Or something along those lines... Basically just want to convey as part of the link text that it'll download the image (rather than redirect to the site), and also provide something a bit more than just "this image"